samples/snippets/src/main/java/com/example/bigquerystorage/WriteToDefaultStream.java

```diff
@@ -151,7 +153,10 @@ public void initialize(TableName parentTable)
       // For more information about JsonStreamWriter, see:
       // https://googleapis.dev/java/google-cloud-bigquerystorage/latest/com/google/cloud/bigquery/storage/v1/JsonStreamWriter.html
       streamWriter =
-          JsonStreamWriter.newBuilder(parentTable.toString(), BigQueryWriteClient.create()).build();
+          JsonStreamWriter.newBuilder(parentTable.toString(), BigQueryWriteClient.create())
```

Comment:
our example code create the client by "BigQueryWriteClient.create()" does that means we are not passing any keepalive setting?

Reply:
I added as a default setting. If I use a new transport in the sample, then the default setting will be overwritten.

I think in most cases, users doesn't need to reset the transport settings, our default is covering all that's needed (being set by Dataflow).